src/services/package-manager/package-service.ts

```diff
@@ -270,12 +271,11 @@ class PackageService {
         return importedPackages.includes(version);
     }
 
-    private async createBodyForImport(filename: string): Promise<object> {
-        return {
-            formData: {
-                package: await fs.createReadStream(filename, {encoding: null})
-            },
-        }
+    private createBodyForImport(filename: string): FormData {
```

Comment:
Why did we change this?

Reply:
The `postFile` method of the `httpService` was generalized as it was specific to this method, so the change was needed to adapt to the new implementation.